xls/examples/parameterized/BUILD

```diff
@@ -0,0 +1,105 @@
+load("//xls/build_rules:xls_utilities.bzl", "proto_to_dslx")
+load("@bazel_skylib//rules:common_settings.bzl", "string_flag")
+load(
+    "//xls/build_rules:xls_build_defs.bzl",
+    "xls_dslx_ir",
+    "xls_dslx_library",
+    "xls_ir_opt_ir",
+    "xls_ir_verilog",
+)
+
+proto_library(
+    name = "proto_schema_lib",
+    srcs = [":param.proto"],
+)
+
+# Parameters for the narrow variant of the design
+proto_to_dslx(
+    name = "parameters_narrow",
```

Comment:
if we used the same output path for both would it create a conflict with bazel?

Reply:
Yes, that's correct